db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -2810,6 +2863,84 @@ class NonBatchedOpsStressTest : public StressTest {
     return true;
   }
 
+  // Compared to VerifyOrSyncValue, VerifyValueRange takes in a
```

Comment:
I thought about adding this functionality into `VerifyOrSyncValue` but that would make the method signature and the implementation even more complicated